lib/chef-dk/command/verify.rb

```diff
@@ -79,8 +79,14 @@ def components
         # We can't run it right now since graphviz specs are included in the
         # test suite by default. We will be able to switch to that command when/if
         # Graphviz is added to omnibus.
-        c.unit_test { sh("bundle exec rspec --color --format progress spec/unit --tag ~graphviz") }
-        c.integration_test { sh("bundle exec cucumber --color --format progress --tags ~@no_run --tags ~@spawn --tags ~@graphviz --strict") }
+        c.unit_test do
+          sh("bundle install")
```

Comment:
Having the smoke tests around should give us a little more confidence that the `bundle install` isn't hiding bugs (simple example, code does `require "foo"` in lib/ but `foo` is listed as a dev dep). Is there anything more we could do to increase the confidence here?